pkg/deploy/deploy.go

```diff
@@ -65,7 +65,7 @@ var (
 	lock                         sync.Mutex
 	concurrentDeploymentsLimiter *rest.ConcurrentRequestLimiter
 
-	skipError = errors.New("skip error")
+	errSkip = errors.New("skip error")
```

Comment:
I probably would have called this `skipErr`,  but it is fine

Reply:
`skipErr` is actually detected by the linter. We can also decide to disable this rule
https://staticcheck.dev/docs/checks/#ST1012
![image](https://github.com/user-attachments/assets/9dc5eb5d-e626-4d7f-aa9c-6a32bc468a9b)